sql/npc_list.sql

```diff
@@ -734,11 +734,13 @@ INSERT INTO `npc_list` VALUES (16822477,'_0b0','Rusty Lever',0,173.021,5.666,-46
 INSERT INTO `npc_list` VALUES (16822478,'_0b1','Rusty Lever',0,196.979,5.666,-71.369,1,50,50,9,0,0,0,4099,0x0200000000000000000000000000000000000000,0,'COP',0);
 INSERT INTO `npc_list` VALUES (16822479,'_0b2','Rusty Lever',0,186.589,13.668,-80.186,1,50,50,9,0,0,0,4099,0x0200000000000000000000000000000000000000,0,'COP',0);
 INSERT INTO `npc_list` VALUES (16822480,'_0b3','DROPFLOOR_A',0,175.003,12.000,-60.009,1,50,50,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
-
+INSERT INTO `npc_list` VALUES (16822481,'_0b6','NPC',0,175.000,16.299,-66.980,1,50,50,9,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
 INSERT INTO `npc_list` VALUES (16822482,'_0b4','DROPFLOOR_B',0,181.003,12.000,-60.009,1,50,50,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
-
+INSERT INTO `npc_list` VALUES (16822483,'_0b7','NPC',0,181.003,16.299,-66.980,1,50,50,9,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
 INSERT INTO `npc_list` VALUES (16822484,'_0b5','DROPFLOOR_C',0,187.003,12.000,-60.009,1,50,50,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
-
+INSERT INTO `npc_list` VALUES (16822485,'_0b8','NPC',0,187.003,16.299,-66.980,1,50,50,9,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16822486,'_0b9','     ',0,181.003,16.299,-66.980,1,40,40,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
+INSERT INTO `npc_list` VALUES (16822487,'_0ba','     ',0,175.000,16.299,-66.980,1,40,40,8,0,0,0,6147,0x0200000000000000000000000000000000000000,0,'COP',0);
```

Comment:
could you tag these that we couldn't find them in an existing cap? thanks for PR'ing!

Reply:
Got that noted and also added you as a co-author on the commit since you helped me discover most of this!

Reply:
well thank you <3